libraries/protocol/include/graphene/protocol/asset_ops.hpp

```diff
@@ -103,6 +104,7 @@ namespace graphene { namespace protocol {
 
       struct ext
       {
+         fc::optional<uint16_t> margin_call_fee_ratio; // BSIP 74
```

Comment:
Although this time it doesn't matter since all parameters are newly added, we should always add new parameters to the end of the structs used by `extensions`, otherwise it breaks consensus.

By the way, need new code in the cpp file to validate the new parameter.